src/libraries/System.Globalization/tests/CompareInfo/CompareInfoTestsBase.cs

```diff
@@ -45,5 +45,6 @@ public class CompareInfoTestsBase
                                                               s_invariantCompare.Compare("\u3060", "\uFF80\uFF9E", CompareOptions.IgnoreKanaType | CompareOptions.IgnoreWidth | CompareOptions.IgnoreCase) == 0;
 
         protected static bool IsNotWindowsKanaRegressedVersionAndNotHybridGlobalizationOnWasm() => !PlatformDetection.IsHybridGlobalizationOnBrowser && IsNotWindowsKanaRegressedVersion();
+        protected static bool IsNotWindowsKanaRegressedVersionAndNotHybridGlobalization() => IsNotWindowsKanaRegressedVersionAndNotHybridGlobalizationOnWasm() && !PlatformDetection.IsHybridGlobalizationOnOSX;
```

Comment:
```suggestion
        protected static bool IsNotWindowsKanaRegressedVersionAndNotHybridGlobalization() => IsNotWindowsKanaRegressedVersion() && PlatformDetection.IsNotHybridGlobalization();
```
Similar to the comment in PlatformDetection.cs

Since we have `IsNotHybridGlobalizationOnBrowser`, maybe we can update the `!PlatformDetection.IsHybridGlobalizationOnBrowser` to be `PlatformDetection.IsNotHybridGlobalizationOnBrowser`.

Also it would be nice to be consistent with the `Browser` vs `Wasm`. Would there be HybridGlobalization on Wasi?

Reply:
In the future maybe there will be `HybridGlobalization` on `WASI`, but now all the implementation in `WASM` is done using `browser API` and there is no `browser API` in `WASI`.

Reply:
Right, I would expect `IsNotWindowsKanaRegressedVersionAndNotHybridGlobalizationOnWasm` in the future to be `!PlatformDetection.IsHybridGlobalizationOnWasm && IsNotWindowsKanaRegressedVersion()` (what I meant by consistent)

Where maybe `PlatformDetection.IsHybridGlobalizationOnWasm = PlatformDetection.IsHybridGlobalizationOnBrowser || PlatformDetection.IsHybridGlobalizationOnWasi`.

So to that pattern `IsNotWindowsKanaRegressedVersionAndNotHybridGlobalizationOnWasm` -> `IsNotWindowsKanaRegressedVersionAndNotHybridGlobalizationOnBrowser` here would be more consistent but thats just a nit and can be addressed when `HybridGlobalization` is extended to `Wasi`.